scripts/style/shellcheck_skip.txt

```diff
@@ -48,6 +48,7 @@ scripts/offline-bundle/image-collector-bundle/import.sh
 scripts/port-forward-ui.sh
 scripts/reference/build-quay-operator-bundles.sh
 scripts/sync-release-branch.sh
+tests/e2e/lib.sh
```

Comment:
Was this change required because you stopped doing `ci_export`-s?

Reply:
Maybe? Style-check asked for this after my latest changes so I added it, but I could run some experiments to see what exactly is causing style-check to want to add this to the ignore. Notably there are already a lot of functions marked `shellcheck disable X` in `e2e/lib.sh`. Right now I'm trying to get compatibility tests to pass again in the first place.